### PR TITLE
Add user session cookie in oauth2 login call 

### PR DESCRIPTION
**Pull request by farhan**
Sending all user session cookies in login call. Mobile apps use this api to fetch user session cookies.

https://edlyio.atlassian.net/browse/EDLY-1723

### How to test:
Please refer to Jira story for more details and testing curl requests.

### CODE REVIEW

**Review by asadmanzoor93 — CHANGES_REQUESTED** [review]
Kindly check if related unit tests need to be updated.

**Comment by ziafazal** [comment]
@farhan I have a question. 
Why edX is not doing this? why we need to this for our mobile app?

**Comment by farhan** [comment]
> @farhan I have a question.
> Why edX is not doing this? why we need to this for our mobile app?

@ziafazal We are having multisite authentication issues on x-block web page fragments needed on mobile to render. For that we need `edly-user-info` cookie to set in mobile native browsers. `edly-user-info` cookie is the only need we have now which is not the requirement of edx. 
To avoid maintenance we can also send all the session cookies to mobile apps which we are doing in this PR. Could there be any negative possible effect of that?
edX might sending the cookies just on demand.

**Comment by farhan** [comment]
Adding more details of Question "Why do we need cookies for mobile apps?"
**Answer:**
We need these cookies because we are rendering the web-fragments in the mobile for example HTML xblock units. We need to set the required session cookies in the mobile native browser before hitting the url.
We don't need the cookies for server calls. Server calls just needs authorization token to get identified.

Currently edx returns around 5 cookies in `oauth2/login/` api.
After this PR all following cookies will be returned.
```
openedx-language-preference
edx-user-info
edxloggedin
edly-user-info
edx-jwt-cookie-header-payload
edx-jwt-cookie-signature
sessionid
csrftoken
edx-jwt-refresh-cookie
AWSALBCORS
AWSALB
```

**Comment by farhan** [comment]
@asadmanzoor93 @iam-mhaseeb  Tests are passing successfully.

https://github.com/edly-io/edx-platform/blob/master/openedx/core/djangoapps/auth_exchange/tests/test_views.py#L161-L200

Screen Shot 2020-08-25 at 5 42 23 PM

**Review by asadmanzoor93 — APPROVED** [review]
👍

File: openedx/core/djangoapps/auth_exchange/tests/test_views.py [hunk 1]
```diff
@@ -213,10 +214,14 @@ def test_invalid_token(self):
...
     def test_dot_password_grant_supported(self):
         access_token = self._create_dot_access_token(grant_type='password')
-
-        self._verify_response(access_token, expected_status_code=204, expected_cookie_name='sessionid')
+        self._verify_response(access_token, expected_status_code=204, expected_cookie_names=['sessionid'])
         self.assertEqual(int(self.client.session['_auth_user_id']), self.user.id)
 
+    def test_expected_cookies_exist(self):
+        access_token = self._create_dot_access_token(grant_type='password')
+        self._verify_response(access_token, expected_status_code=204,
```
Comment (zubair-arbi) [inline openedx/core/djangoapps/auth_exchange/tests/test_views.py:222]: update indentation:
```python
self._verify_response(
    access_token, 
    expected_status_code=204,
    expected_cookie_names=['edx-user-info', 'edly-user-info']
)
```

**Review by zubair-arbi — APPROVED** [review]
One suggestion, otherwise changes looks good. 👍

**Review by iam-mhaseeb — APPROVED** [review]
LGTM 👍

File: openedx/core/djangoapps/auth_exchange/tests/test_views.py (hunk 1 above)
Comment (farhan) [reply openedx/core/djangoapps/auth_exchange/tests/test_views.py]: @zubair-arbi done. Going to squash the commits.

**Pull request merged**